### PR TITLE
rename `whitelist_url` to `allow_url`

### PR DESCRIPTION
**Pull request by samandmoore**
i think this is maybe the last change before creating a 1.0.0. 

we've been in production with this library for a few years and we've pushed it pretty hard.

/domain @smudge @jmileham 
/no-platform

i'm making this as a breaking change because we're not at 1.0.0 yet and it's a straightforward upgrade

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ This will drop a new file in your config directory.
 # WebValve.register FakeBank
 # WebValve.register FakeExample, url: 'https://api.example.org'
 #
-# # whitelist urls
+# # add urls to allowlist
 #
-# WebValve.whitelist_url 'https://example.com'
+# WebValve.add_url_to_allowlist 'https://example.com'
```
Comment (samandmoore) [inline README.md]:
```suggestion
# WebValve.allowlist_url 'https://example.com'
```

**Comment by smudge** [comment]
<< domain LGTM -- but why not `allow_url` for the config method?

**Comment by samandmoore** [comment]
> << domain LGTM -- but why not `allow_url` for the config method?

ah yes, let's do that!

**Comment by smudge** [comment]
@samandmoore any idea what @nanda-prbot's going on about?

**Comment by samandmoore** [comment]
Nope. I guess there's a background task that is stuck and keeps retrying.

**Pull request merged**